tests/test_init.py

```
@@ -834,9 +840,11 @@ async def test_async_setup_entry_awesomeversion_exception(
     # monkeypatch AwesomeVersion to a class that raises on comparison
     class DummyAV:
         def __init__(self, v):
+            """Store the version string used by the comparison stub."""
             self.v = v
 
         def __lt__(self, other):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__lt__`

(ANN204)